src/routes/console/project-[project]/functions/function-[function]/deployment-[deployment]/+page.ts

```diff
@@ -2,10 +2,17 @@ import { sdk } from '$lib/stores/sdk';
 import { Dependencies } from '$lib/constants';
 import type { PageLoad } from './$types';
 
-export const load: PageLoad = async ({ params, depends }) => {
+export const load: PageLoad = async ({ params, depends, parent }) => {
+    const data = await parent();
```

Comment:
```suggestion
    const { deployment } = await parent();
```
It's easier to know what we want from the parent, and it will be highlighted as an error in the code editor if the parent does not return it

Reply:
done, but function is a reserved keyword, so renamed func